### ISSUE TITLE
Put on npm?

### ISSUE THREAD

**Issue by sethreidnz** [scottdorman/mdl2-icons#1]
Would you potentially put this on npm so we can download and manage it that way?

**Comment by scottdorman** [comment]
Yes, this will be up on NPM. Right now, I'm working on adding a lot of additional icons, removing duplicates, and making sure things will be stable between releases.

**Comment by theBull** [comment]
@scottdorman will this also have an MIT license?

**Comment by scottdorman** [comment]
@theBull I just added an MIT license file to the project. @sethreidnz I will be putting it up on NPM this week. Sorry for the delays.
